This patch removed some invalid lines from the man page and adds some
information about -- (closes #497985)
diff -Naur a/id3v2.1 b/id3v2.1
--- a/id3v2.1	2002-02-14 20:35:31.000000000 +0100
+++ b/id3v2.1	2009-06-07 20:04:49.213540395 +0200
@@ -6,11 +6,9 @@
 .RB [
 .I OPTION
 .RB ]
-...
 .RB [
 .I FILE
 .RB ]
-...
 .br
 .SH OPTIONS
 .TP
@@ -65,6 +63,15 @@
 .B \-T, \-\-track num/num
 Set the track number/(optional) total tracks
 
+.SH NOTE
+You can set the value for any id3v2 frame by using '--' and then frame id
+.TP
+For example:
+
+.B id3v2 --TIT3 "Monkey!" file.mp3
+.TP
+would set the "Subtitle/Description" frame to "Monkey!".
+
 .SH SEE ALSO
 id3tag(1), id3info(1), id3convert(1)
 .SH AUTHOR
